portfolio.js

```diff
@@ -1,3 +1,204 @@
+  body.prepend(PopUp);
+  if (window.innerWidth < 768) {
+    const btMobWhite = document.querySelector('.btMobWhite');
+    btMobWhite.addEventListener('click', ClosePopUp);
+  } else {
+    const btMobDesktop = document.querySelector('.btMobDesktop');
+    btMobDesktop.addEventListener('click', ClosePopUp);
+  }
+} */
+
+document.addEventListener('DOMContentLoaded', () => {
+  const string = createStrProjects(countProj);
+  document.querySelector('#portf').innerHTML = string; // here projetcs loaded with ID buttons.
+
+  /* PopUp Mobile & Desktop */
+
+  const button1 = document.querySelector('#Button1');
+  const button2 = document.querySelector('#Button2');
+  const button3 = document.querySelector('#Button3');
+  const button4 = document.querySelector('#Button4');
+  const button5 = document.querySelector('#Button5');
+  const button6 = document.querySelector('#Button6');
+
+  button1.addEventListener('click', () => {
+    const body = document.querySelector('.body');
+    const PopUp = document.createElement('div');
+    PopUp.classList.add('MobilePopUp');
+    PopUp.innerHTML = createStrProjectMob('button1');
+    body.prepend(PopUp);
+    if (window.innerWidth < 768) {
+      const btMobWhite = document.querySelector('.btMobWhite');
+      btMobWhite.addEventListener('click', ClosePopUp);
+    } else {
+      const btMobDesktop = document.querySelector('.btMobDesktop');
+      btMobDesktop.addEventListener('click', ClosePopUp);
+    }
+  });
+
+  button2.addEventListener('click', () => {
+    const body = document.querySelector('.body');
+    const PopUp = document.createElement('div');
+    PopUp.classList.add('MobilePopUp');
+    PopUp.innerHTML = createStrProjectMob('button2');
+    body.prepend(PopUp);
+    if (window.innerWidth < 768) {
+      const btMobWhite = document.querySelector('.btMobWhite');
+      btMobWhite.addEventListener('click', ClosePopUp);
+    } else {
+      const btMobDesktop = document.querySelector('.btMobDesktop');
+      btMobDesktop.addEventListener('click', ClosePopUp);
+    }
+  });
+
+  button3.addEventListener('click', () => {
+    const body = document.querySelector('.body');
+    const PopUp = document.createElement('div');
+    PopUp.classList.add('MobilePopUp');
+    PopUp.innerHTML = createStrProjectMob('button3');
+    body.prepend(PopUp);
+    if (window.innerWidth < 768) {
+      const btMobWhite = document.querySelector('.btMobWhite');
+      btMobWhite.addEventListener('click', ClosePopUp);
+    } else {
+      const btMobDesktop = document.querySelector('.btMobDesktop');
+      btMobDesktop.addEventListener('click', ClosePopUp);
+    }
+  });
+
+  button4.addEventListener('click', () => {
+    const body = document.querySelector('.body');
+    const PopUp = document.createElement('div');
+    PopUp.classList.add('MobilePopUp');
+    PopUp.innerHTML = createStrProjectMob('button4');
+    body.prepend(PopUp);
+    if (window.innerWidth < 768) {
```

Comment:
- [ ] I think it would be better to create a function that takes the required parameters to perform this rule, instead of writing the same code 6 times, because this is recommended in the JavaScript best practices [here](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md#:~:text=simplicity.-,DRY,-.) as do not repeat yourself :recycle: